### PR TITLE
[19] Users can see a start page for the planning journey

### PR DESCRIPTION
**Pull request by tahb**
## Changes in this PR

The service throws away the home page that comes from the Rails Template and replaces it with most (see commit) of the copy from the alpha. [The page on the prototype can be found here](https://dfe-hubs-prototype.herokuapp.com/sprint-one/support-start-page).

We create a new static page with HighVoltage rather than a controller since it seems a better fit for static content.

We have a card focused on addressing the styling so this work doesn't address styling considerations.

### Before
Screenshot 2020-10-26 at 14 16 23

### After

Screenshot 2020-10-26 at 12 24 10

### CODE REVIEW

File: app/views/pages/planning_start_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<h1><%= I18n.t("planning.start_page.page_title") %></h1>
```
Comment (jacksonj04) [inline app/views/pages/planning_start_page.html.erb:1]: I think this also needs including in the `<title>` for the page

**Review by jacksonj04 — CHANGES_REQUESTED** [review]
One tweak to get the title in the `<title>` element, otherwise looks like a solid set of words.

File: app/views/pages/planning_start_page.html.erb (hunk 1 above)
Comment (tahb) [reply app/views/pages/planning_start_page.html.erb:1]: Ah yes, good spot 👍

**Comment by tahb** [comment]
@jacksonj04 found another small thing on my way past https://github.com/DFE-Digital/buy-for-your-school/pull/4/commits/33bfa5b94abf7c65519186d528342f6a4296913d

**Review by jacksonj04 — APPROVED** [review]
LGTM 🚀

**Pull request merged**